app/controllers/api/base_controller.rb

```diff
@@ -82,10 +82,10 @@ def not_found(options = nil)
       when Hash
         not_found_message.merge! options
       else
-        render_error 'not_found', :status => :not_found and return false
+        render_error('not_found', :status => :not_found) && (return false)
```

Comment:
Hmm, this is what rubocop's autocorrect did. I am tempted to just break these `and return`s into two lines.